docker-compose.production.yaml

```diff
@@ -53,6 +53,11 @@ services:
     command: ["sh", "-c", "redis-server --requirepass \"$REDIS_PASSWORD\""]
     env_file: .env
     volumes:
-      - /data/redis:/data
+      - redis_data:/data
     ports:
-      - '127.0.0.1:6379:6379'
+      - '6379:6379'
```

Comment:
`127.0.0.1:6379:6379`- This is needed because we don't want to expose this port outside localhost for a production deployment.